compressor/offline/django.py

```diff
@@ -26,7 +26,7 @@ def handle_extendsnode(extendsnode, block_context=None, original=None):
                   extendsnode.nodelist.get_nodes_by_type(BlockNode))
     block_context.add_blocks(blocks)
 
-    context = Context(settings.COMPRESS_OFFLINE_CONTEXT)
+    context = Context(dict(settings.COMPRESS_OFFLINE_CONTEXT))
```

Comment:
Ah, forgot that context was used here when I merged 693464a1c278855747c1b5c96c48dcdd4c5f9cab, so I guess we'll need to refactor things a bit...